admin_interface/static/admin_interface/css/tabbed-changeform.css

```diff
@@ -33,3 +34,8 @@
 .admin-interface .tabbed-changeform-tabcontent.active {
     display: block;
 }
+
+.admin-interface #rest-of-the-tabber {
```

Comment:
Could you change this from id to class and rename it to `.tabbed-changeform-tabs-remaining-space` please?

Reply:
updated